src/libraries/tests.proj

```diff
@@ -27,10 +27,11 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.ComponentModel.TypeConverter\tests\System.ComponentModel.TypeConverter.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Data.Common\tests\System.Data.Common.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Diagnostics.DiagnosticSource\tests\TestWithConfigSwitches\System.Diagnostics.DiagnosticSource.Switches.Tests.csproj" />
+
+    <!-- Mono-Browser ignores runtimeconfig.template.json (e.g. for these two it has "System.Globalization.Invariant": true) -->
```

Comment:
So the default mode will now be use, ICU? Should we make the tests set the env var before they start running?

Reply:
We should just use csproj setting, it will be transferred to correct InvariantSetting for the platform and save us messing up with template.json

Reply:
If we add the relevant logic to WasmAppBuilder it can send the invariant mode flag into the mono-config js bundle where the loader will read it and configure things at runtime

Reply:
What do you think needs configuration?

Reply:
> We should just use csproj setting, it will be transferred to correct InvariantSetting for the platform and save us messing up with template.json

We would need to publish the tests trimmed right for the Linker to replace the GlobalizationMode.Invariant property to true?

Reply:
It set the appcontext settings too

Reply:
Yeah, but I thought that didn't work for wasm for some reason. I thought setting those project settings would just add them to the `runtimeconfig.json` which is not supported by wasm yet.